charts/prometheus-pushgateway/templates/_helpers.tpl

```diff
@@ -119,6 +119,10 @@ serviceAccountName: {{ include "prometheus-pushgateway.serviceAccountName" . }}
 {{- with .Values.priorityClassName }}
 priorityClassName: {{ . | quote }}
 {{- end }}
+{{- with .Values.hostAliases }}
+hostAliases:
+{{- toYaml . | nindent 8 }}
```

Comment:
We have to consider indentation of the helper template `prometheus-pushgateway.podSpec` in the deployment template itself:

```suggestion
{{- toYaml . | nindent 2 }}
```

Reply:
good catch, thanks